install/terraform/modules/helm/helm.tf

```
@@ -153,6 +153,26 @@ resource "helm_release" "agones" {
     name  = "agones.featureGates"
     value = var.feature_gates
   }
+  
+  set {
+    name  = "agones.featureGates"
+    value = var.feature_gates
+  }
+
+  set {
+    name  = "gameservers.namespaces"
+    value = var.gameserver_namespaces
+  }
+
+  set {
+    name  = "gameservers.minport"
```

Should it be the same as in `values.yaml`?
```suggestion
    name  = "gameservers.minPort"
```